### PR TITLE
Document Release notes for SBO 1.2

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
[RHDEVDOCS-4327](https://issues.redhat.com/browse/RHDEVDOCS-4327): Release Notes, New Features and Known Issues for SBO 1.2

- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: enterprise-`4.11` and later
- **JIRA issues**: [RHDEVDOCS-4327](https://issues.redhat.com/browse/RHDEVDOCS-4327)
- **Preview pages**: https://50453--docspreview.netlify.app/openshift-enterprise/latest/applications/connecting_applications_to_services/sbo-release-notes.html
- **SME Review**: Completed by @baijum, @dperaza4dustbit, @sadlerap
- **QE review**: Completed by @pmacik 
- **Peer-review**: Completed by @rolfedh

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
Hi, @dperaza4dustbit, @sadlerap, @fbm3307, and @pmacik: PTAL at the PR and provide feedback. Thanks!

File: modules/sbo-release-notes-1-2.adoc
Comment (dperaza4dustbit) [inline modules/sbo-release-notes-1-2.adoc:54]: I think we need to specify what correct or incorrect is here

**Review by dperaza4dustbit — COMMENTED** [review]
One suggestion added

File: modules/sbo-release-notes-1-2.adoc
Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-2.adoc:54]: Hi, @sadlerap: PTAL at David's comment. Please provide the required inputs. Thanks!

**Comment by Srivaralakshmi** [comment]
Note to reviewers: The content for Known Issue [APPSVC-1062](https://issues.redhat.com/browse/APPSVC-1062) exists since the last 2 releases and is already reviewed. Please review the rest of the content. Thanks!

File: modules/sbo-release-notes-1-2.adoc
Comment (rolfedh) [inline modules/sbo-release-notes-1-2.adoc:20]: 💡 RedHat.Usage 15(124-130) Verify your use of 'due to' with the word usage guidelines.
💡 RedHat.SimpleWords 15(256-262) Use simple language. Consider using 'change' rather than 'modify'.

File: modules/sbo-release-notes-1-2.adoc
Comment (rolfedh) [reply modules/sbo-release-notes-1-2.adoc:54]:
```suggestion
* Currently, when you modify the `ClusterWorkloadResourceMapping` resources, the {servicebinding-title} does not implement the correct behavior. As a workaround, perform the following steps:
```

**Review by rolfedh — COMMENTED** [review]
One optional style suggestion and one comment that will probably become obsolete when you update the content around line 54. Please ping me to approve this PR when you're done.

**Comment by pmacik** [comment]
appart from changes suggested by @rolfedh
/lgtm

File: modules/sbo-release-notes-1-2.adoc
Comment (sadlerap) [reply modules/sbo-release-notes-1-2.adoc:54]: Maybe something like this:
```suggestion
* Currently, when you modify the `ClusterWorkloadResourceMapping` resources, the {servicebinding-title} does not implement correct behavior. When modified, the specification requires that workloads have their binding data de-projected using the old `ClusterWorkloadResourceMapping` resource, then re-projected using the new resource. As a workaround, perform the following steps:
```

Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-2.adoc:54]: @sadlerap According to the specification when you change the `ClusterWorkloadResourceMapping` resources, the {servicebinding-title} must use the previous version of the `ClusterWorkloadResourceMapping` resource to remove the binding data that was getting projected till now. Currently, when you change the `ClusterWorkloadResourceMapping` resources, the {servicebinding-title} uses the latest version of the `ClusterWorkloadResourceMapping` resource to remove the binding data. As a result, {servicebinding-title} might remove the binding data incorrectly. As a workaround, perform the following steps:

Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-2.adoc:54]: Done.

**Comment by Srivaralakshmi** [comment]
@dperaza4dustbit, @sadlerap, and @rolfedh: PTAL at the revised change to the Known issue (APPSVC-1102) and provide your feedback. Thanks!

File: modules/sbo-release-notes-1-2.adoc
Comment (rolfedh) [inline modules/sbo-release-notes-1-2.adoc:54]:
```suggestion
* According to the specification, when you change the `ClusterWorkloadResourceMapping` resources, {servicebinding-title} must use the previous version of the `ClusterWorkloadResourceMapping` resource to remove the binding data that was being projected until now. Currently, when you change the `ClusterWorkloadResourceMapping` resources, the {servicebinding-title} uses the latest version of the `ClusterWorkloadResourceMapping` resource to remove the binding data. As a result, {the servicebinding-title} might remove the binding data incorrectly. As a workaround, perform the following steps:
```

Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-2.adoc]: Done

**Review by rolfedh — APPROVED** [review]

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

File: modules/sbo-release-notes-1-2.adoc
Comment (Srivaralakshmi) [inline modules/sbo-release-notes-1-2.adoc:20]: @dperaza4dustbit and @pmacik: PTAL at this newly added Known Issue and provide feedback. 

Please provide the APPSVC JIRA link to this issue. I will add it here. Thanks!

Comment (pmacik) [reply modules/sbo-release-notes-1-2.adoc:20]: Hi @Srivaralakshmi the JIRA issue is [APPSVC-1200](https://issues.redhat.com/browse/APPSVC-1200).

Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-2.adoc:20]: @pmacik Thanks for the issue link. Does the RN Text look good? If yes, then pls provide your LGTM in the PR and I will add it to the JIRA.

**Comment by pmacik** [comment]
LGTM

File: modules/sbo-release-notes-1-2.adoc
Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-2.adoc:20]: Done.

**Comment by Srivaralakshmi** [comment]
Hi, @gabriel-rh The Release Notes for SBO 1.2 needs to be merged today. The PR has received all the approvals from SMEs and Rolfe. However, there is a last-minute addition to the Known Issue - just one.
A quick review and approval from you would be helpful. I can then send a merge request.

File: modules/sbo-release-notes-1-2.adoc
Comment (gabriel-rh) [inline modules/sbo-release-notes-1-2.adoc:20]: Maybe don't start the sentence with "But ", and just say...

"With low usage, the memory.."

or maybe better:

"With low usage, however, the memory.."

**Review by gabriel-rh — COMMENTED** [review]
Looks good - just a minor suggestion not to start the sentence with "But ", apart from that everything looks good to me!

File: modules/sbo-release-notes-1-2.adoc
Comment (Srivaralakshmi) [reply modules/sbo-release-notes-1-2.adoc]: Done.

**Comment by Srivaralakshmi** [comment]
@gabriel-rh I have implemented your suggestion. PTAL and provide your LGTM. Thanks!

**Review by gabriel-rh — APPROVED** [review]
/lgtm

**Pull request merged**
